src/avitab/apps/MapApp.cpp

```diff
@@ -63,7 +63,7 @@ MapApp::MapApp(FuncsPtr funcs):
     chooserContainer->centerInParent();
     chooserContainer->setVisible(false);
 
-    setMapSource(MapSource::OPEN_TOPO);
+    setMapSource(MapSource::ONLINE_TILES, true);

```

Comment:
When we construct the MapApp object, we now call the `setMapSource` function with `init` set to `true`.